### DISCUSSION TITLE
iOS Access without CA

### DISCUSSION THREAD

**Discussion by DaCHack** [dani-garcia/vaultwarden#1426] [Q&A]
I am trying to access my vault from a corporate phone. The installed configuration policy does not allow me to add own trusted CA.

Any chance for me to get access with the iOS app for this user without switching to a pure HTTP setup which in turn will not work in Chrome?

also I would like to avoid exposing my server to the internet and running it with a Lets Encrypt certificate. Id rather stay behind Firewall/VPN.

Many Thanks!

**Comment by 0xERR0R** [comment]
I would suggest to use Lets Encrypt with DNS challenge. In this case, you must not expose any ports. I use this setup with DuckDNS free domain and traefik as reverse proxy. Traefik makes the cetrificate magic and I can reach bitwarden_rs over VPN with HTTPS
